### ISSUE TITLE
api.Groups.allProjects Simple Property is always true

### ISSUE THREAD

**Issue by henrysachs** [jdalrymple/gitbeaker#3270]
Description

Node.js version: 18
Gitbeaker version: 38.7.0
Gitbeaker release (cli, node, browser, core, requester-utils): rest
OS & version MacOs
Steps to reproduce

Use the api.Groups.allProjects Method an try to set the simple field to false

Expected behaviour

the property is usable in typescript and if set to false returns more properties

Actual behaviour

property has type true

Possible fixes

make it an optional property a boolean i did this manually:

``` typescript
    allProjects<E extends boolean = false, P extends PaginationTypes = 'offset'>(groupId: string | number, options?: {
        simple: boolean;
        sharedOnly?: boolean;
    } & AllGroupProjectsOptions & PaginationRequestOptions<P> & Sudo & ShowExpanded<E>): 
```

Checklist

[ x ] I have checked that this is not a duplicate issue.
[ x ] I have read the documentation.

**Comment by jdalrymple** [comment]
Will be released by EoD

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`38.8.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/38.8.0) :rocket:
